### ISSUE TITLE
TCP flow table needs to feed into expiration

### ISSUE THREAD

**Issue by rzezeski** [oxidecomputer/opte#3]
The TCP flow table should be consulted when expiring flows in the layer flow tables and unified flow table, conversely the layer/unified flow table TTL should feed into expiring TCP flow table entries.

**Comment by rzezeski** [comment]
Now when a TCP connection transitions to the closed state the flow is removed from the UFT. I did not implement the other way around (UFT expiration deleting TCP flow entries), that will come in a separate issue.

* UFT during active SSH session.

```
rpz@kalm:~/oxidecomputer/opte$ pfexec opteadm dump-uft -p opte1
Unified Flow Table
======================================================================
Inbound Flows [3/8096]
----------------------------------------------------------------------
PROTO  SRC IP           SPORT  DST IP           DPORT  HITS     ACTION                
TCP    10.0.0.10        50299  10.0.0.226       22     3        NAT4                  
UDP    10.0.0.14        42892  255.255.255.255  1900   1                              
UDP    10.0.0.39        57621  10.0.0.255       57621  613                            

Outbound Flows [1/8096]
----------------------------------------------------------------------
PROTO  SRC IP           SPORT  DST IP           DPORT  HITS     ACTION                
TCP    172.30.0.5       22     10.0.0.10        50299  3        NAT4  
```

* Closed the SSH session.

```
localhost:~# 
Connection to 10.0.0.226 closed.
```

* UFT after session is closed.

```
rpz@kalm:~/oxidecomputer/opte$ pfexec opteadm dump-uft -p opte1
Unified Flow Table
======================================================================
Inbound Flows [2/8096]
----------------------------------------------------------------------
PROTO  SRC IP           SPORT  DST IP           DPORT  HITS     ACTION                
ICMP   10.0.0.1         0      10.0.0.226       0      0        NAT4                  
UDP    10.0.0.39        57621  10.0.0.255       57621  613                            

Outbound Flows [0/8096]
----------------------------------------------------------------------
PROTO  SRC IP           SPORT  DST IP           DPORT  HITS     ACTION            
```

**Comment by rzezeski** [comment]
Addressed in fb01edd13197fbc8626f852397ce2090a8502cbe.
